test-suite/shared/src/test/scala/org/scalajs/testsuite/niocharset/CharsetTest.scala

```diff
@@ -67,4 +69,24 @@ class CharsetTest {
 
     assertFalse(Charset.isSupported("this-charset-does-not-exist"))
   }
+
+  @Test def aliases(): Unit = {
+    assertEquals(Charset.forName("UTF-8").aliases(),
+      List("UTF8", "unicode-1-1-utf-8").toSet.asJava)
```

Comment:
Use `Set(...)` instead of `List(...).toSet`. Even better, define a local
```scala
def javaSet[A](elems: A*): java.util.Set[A] = Set(elems: _*).asJava
```
and then use `javaSet("UTF8", "unicode-1-1-utf-8")`.

Reply:
Also, we use 4 spaces indentation when continuing a line. See our coding style at https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#indentation